communication/mavlink_waypoint_handler.cpp

```diff
@@ -503,7 +502,6 @@ void Mavlink_waypoint_handler::init_homing_waypoint()
                         navigation_.takeoff_altitude);
 
     waypoint_count_ = 1;
-    navigation_.set_waiting_at_waypoint(false);
```

Comment:
@mdouglas90 I removed this because I think waypoint_handler should not modify navigation